### PR TITLE
Add Image Input

### LINKED ISSUE #9: add image input
[`add-image-input`](https://github.com/iFixit/Bootstrap-Form-Builder/compare/add-image-input)

### PR DESCRIPTION
**Pull request by davidrans**
See https://github.com/iFixit/ifixit/pull/17090 for more details.

Closes #9

### CODE REVIEW

File: assets/js/templates/snippet/image.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!-- File Button -->
+<div class="control-group">
+  <label class="control-label"
+         for="<%= field_guid %>"><%= label %></label>
+  <button id="<%= field_guid %>" name="<%= field_guid %>">Upload</button>
```
Comment (solflare) [inline assets/js/templates/snippet/image.html]: Do we want the 'btn' class (or any others) like here: https://github.com/iFixit/Bootstrap-Form-Builder/blob/master/assets/js/templates/snippet/button.html#L5?

Comment (sterlinghirsh) [reply assets/js/templates/snippet/image.html]: I'm not sure if we have a means of translating "Upload" but that could be something. I think it only shows up in the edit UI so maybe it doesn't matter much.

**Comment by sterlinghirsh** [comment]
One small translation issue but otherwise CR :zap:

File: assets/js/templates/snippet/image.html (hunk 1 above)
Comment (davidrans) [reply assets/js/templates/snippet/image.html]: Hmm, I don't think we translate any of the text in any of the other snippets. Maybe that's something we should look into doing.

**Pull request merged**